README.md

```diff
@@ -201,241 +52,58 @@ git clone git@github.com:GitHubUser/i18n.git
 cd i18n
 ```
 
```

Comment:
Are we missing step 2? I think we need to clone before loading submodules.

```{suggestion}
2. Clone this repository from your personal account (e.g., GitHubUser). This is your local copy to manage your version of
 the translation files.

cd directory
git clone git@github.com:GitHubUser/i18n.git
cd i18n
```

Reply:
Sorry I can get MD backticks into suggestion mode

Reply:
Actually, it's not missing in the README in my PR. You can see it if you look at the actual file instead of the diff.

Reply:
Ah yes, it was the diff that confused me.